### PR TITLE
Dynamically show the last year in footer

### PR DESCRIPTION
**Pull request by Razer6**
/cc @jbrooksuk

### CODE REVIEW

File: resources/views/partials/footer.blade.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
     <div class="copyright">
         <div class="container">
             <div class="row">
-                <div class="col-lg-12 text-right">&copy; 2016 <a href="https://alt-three.com">Alt Three Services Limited</a>, registered no. 9671573</a></div>
+                <div class="col-lg-12 text-right">&copy; 2014 - {{ date('Y') }} <a href="https://alt-three.com">Alt Three Services Limited</a>, registered no. 9671573</a></div>
```
Comment (GrahamCampbell) [inline resources/views/partials/footer.blade.php]: Please omit the spaces either side of the `-` to be consistent with our license formatting, as discussed in the issue.

Comment (Razer6) [inline resources/views/partials/footer.blade.php]: ok. Should the year be changed to 2015 is discussed in the issue or should it stay 2014?

Comment (GrahamCampbell) [inline resources/views/partials/footer.blade.php]: Technically Alt Three Services Limited has only officially existed since mid-2015.

**Comment by jbrooksuk** [comment]
But the website content and Cachet was 2014.

**Comment by jbrooksuk** [comment]
Thanks @Razer6, appreciate this :)

**Comment by GrahamCampbell** [comment]
> But the website content and Cachet was 2014.

It didn't belong to alt three though, meh.

**Comment by jbrooksuk** [comment]
It belonged to me. "Me" is Alt Three now essentially.

**Pull request merged**